src/helpers/__tests__/highlight-test.js

```diff
@@ -1,6 +1,6 @@
 import highlight from '../highlight';
 
-/* eslint-disable camelcase */
+/* eslint-disable @typescript-eslint/camelcase */
```

Comment:
The associated closing command must also be updated to `@typescript/eslint`. It still is:

```js
/* eslint-enable camelcase */
```